Dockerfile

```diff
@@ -86,6 +83,5 @@ ENV RAILS_ENV=production
 COPY --from=builder $BUNDLE_APP_CONFIG $BUNDLE_APP_CONFIG
 COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/src ./
 COPY --from=builder --chown=vets-api:vets-api /srv/vets-api/clamav/database ../clamav/database
-RUN if [ -d certs-tmp ] ; then cd certs-tmp ; for i in * ; do cp $i /usr/local/share/ca-certificates/${i/pem/crt} ; done ; fi && update-ca-certificates
```

Comment:
leave this for now, remove it after we have confirmed that the change in base stage is working and we have the devops repo builds set to use it

Reply:
Yep, wasn't going to merge this until everything was confirmed on the other side either way.